### PR TITLE
Added ConnectedPoint constructor and Dynamo method.

### PR DESCRIPTION
**Pull request by xRadne**
Add constructors to `ConnectedPoints` class.

Implemented
- Public C# constructor
- Static method `Define` visible in dynamo

**Not** implemented in this PR:
- Grasshopper component

### CODE REVIEW

File: src/Model/Model.cs [hunk 1]
```diff
@@ -2315,6 +2349,21 @@ public static Model ModelAddConnectedLine(Model fdModel, List<ModellingTools.Con
             
         }
 
+        [IsLacingDisabled()]
+        [IsVisibleInDynamoLibrary(true)]
```
Comment (andosca) [inline src/Model/Model.cs:2353]: Add summary to method.

File: src/Releases/RigidityDataType2.cs [hunk 2]
```diff
@@ -19,5 +19,14 @@ public class RigidityDataType2: RigidityDataType1
         public Releases.Rotations Rotations { get; set; }
         [XmlElement("plastic_limit_moments", Order=4)]
         public Releases.PlasticityType3d PlasticLimitMoments { get; set; }
+
+        public RigidityDataType2() : base()
```
Comment (andosca) [inline src/Releases/RigidityDataType2.cs:23]: Have you tried this with regard to serialization/deserialization? 

I'm not sure how System.Xml.Serialization interprets constructor inheritance.

File: src/Releases/RigidityDataType3.cs [hunk 3]
```diff
@@ -44,27 +44,22 @@ public double Friction
         /// <summary>
         /// Parameterless constructor for serialization.
         /// </summary>
-        private RigidityDataType3()
+        private RigidityDataType3() : base()
```
Comment (andosca) [inline src/Releases/RigidityDataType3.cs:47]: Same as comment on RigidityDataType2.

File: src/Releases/RigidityDataType2.cs (hunk 2 above)
Comment (xRadne) [reply src/Releases/RigidityDataType2.cs:23]: Works just fine! But unnecessary and it's probably more clear to leave it empty as it was

**Comment by xRadne** [comment]
✨ Implemented cosmetic changes according to review, merging!

**Comment by andosca** [comment]
Nice! ✨

**Pull request merged**